src/Vector3.h

```diff
@@ -121,11 +126,7 @@ class Vector3
 	{
 		// Perform a bitwise comparison of the contents - we want to know whether this object is exactly equal
 		// To perform EPS-based comparison, use the EqualsEps() function
-		return (
-			(memcmp(&x, &a_Rhs.x, sizeof(x)) == 0) &&
-			(memcmp(&y, &a_Rhs.y, sizeof(y)) == 0) &&
-			(memcmp(&z, &a_Rhs.z, sizeof(z)) == 0)
-		);
+		return !((x != a_Rhs.x) || (y != a_Rhs.y) || (z != a_Rhs.z));
```

Comment:
How is a memcmp unsafer?

Unless you know of a way to explicitly turn off the warning for this specific instance (and not for the rest of the code), then make the change, otherwise please change this back to the original code. We want as few warnings as possible.